syntax/swift.vim

```diff
@@ -150,6 +150,7 @@ syntax keyword swiftKeywords
       \ required
       \ rethrows
       \ return
+      \ #selector
```

Comment:
Yea I guess that's a good point. I keep thinking of these things as "preprocessor like" but I'm not sure how we should categorize them. As they add more `#*` things we can revisit this.